.vehicle_profiles/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "merge all the different json files",
   "main": "merge.js",
   "scripts": {
-    "build": "node merge.js",
+    "build": "node merge.js; node buildSupportedVehiclesList.js",
```

Comment:
Would prefer it to just run at the end of `merge.js` instead of getting called seperatly. Theres going to end up being more and more that gets done, so just wanting to keep 1 entry file